### ISSUE TITLE
[BUG]: Cannot Sign up with GitHub OAuth Provider

### ISSUE THREAD

**Issue by MohamedHassan499** [activepieces/activepieces#4580]
**Describe the bug**
I tried for the first time to sign up to the app, note that I had to refresh it once for it to work, using GitHub Auth Provider, but then I didn't get redirected to anything. 

**To Reproduce**
1. Go to [this page](https://cloud.activepieces.com/sign-up)
2. Choose `Sign Up With GitHub`.
3. You'll notice nothing will happen, even If you go to [activepieces.com](https://www.activepieces.com/), you'll receive the Nuxt error page (Refer to the second screenshot).
4. When I chose `Sign Up With Google`, nothing happened again. 

**Expected behavior**
- Signed up successfully and redirected to  [cloud.activepieces.com](https://www.cloud.activepieces.com/)

**Screenshots**
![image](https://github.com/activepieces/activepieces/assets/50884619/0f07c7c6-2646-4815-9762-0f913a95234e)
![image](https://github.com/activepieces/activepieces/assets/50884619/6f4aa2ff-1ffd-44e1-8277-e5bcfcf04824)

**Comment by abuaboud** [comment]
Hi @MohamedHassan499 

Can you please try again now? do you still have the issue?

**Comment by MohamedHassan499** [comment]
Hello @abuaboud
The issue persists, please have a look at the response here coming from your API when I tried to sign up again with GitHub.

Response: 
```typescript
{
    "code": "EXISTING_USER",
    "params": {
        "email": "sci.mohamedhassan2018@alexu.edu.eg",
        "platformId": "NgixMLyPUxy1ZgCcxw5cM"
    }
}
```

![image](https://github.com/activepieces/activepieces/assets/50884619/099c698d-57e9-4203-b8c4-30e71fbd9d4d)

**Comment by abuaboud** [comment]
I think I found it, It's due to your github profile doesn't have name 

Can you give it another try? It should be fixed in https://github.com/activepieces/activepieces/pull/4593 and released

**Comment by abuaboud** [comment]
I am closing this issue, Please reopen if it still persist.

@allcontributors please add @MohamedHassan499 for bug

**Comment by MohamedHassan499** [comment]
Correct, having an unassigned name caused the issue. It works now. Thanks for fixing it :)
